spring-cloud-contract-verifier/src/main/groovy/org/springframework/cloud/contract/verifier/builder/XmlBodyVerificationBuilder.java

```diff
@@ -45,7 +45,9 @@ void addXmlResponseBodyCheck(BlockBuilder blockBuilder, Object responseBody,
 	private void addXmlProcessingLines(final BlockBuilder blockBuilder,
 			String responseString) {
 		Arrays.asList(
-				"DocumentBuilder documentBuilder = DocumentBuilderFactory.newInstance().newDocumentBuilder()",
+				"DocumentBuilderFactory builderFactory = DocumentBuilderFactory.newInstance()",
```

Comment:
I don't remember if you've already added those but have you added necessary imports for these classes?

Reply:
I checked and the imports are already there.